core/src/navigation/LeftNav.html

```diff
@@ -56,7 +63,11 @@
           <li class="fd-nested-list__item">
             <a
               href="{getRouteLink(node)}"
-              title="{$getTranslation(node.label)}"
+              title="{
+                (node.tooltipText && node.tooltipText.length) ?
+                $getTranslation(node.tooltipText) :
+                (node.tooltipText === false) ? '' : $getTranslation(node.label)
```

Comment:
`(node.tooltipText === false) ? '' : $getTranslation(node.label)`
Worked fine for me, but I think that the original [Issue](https://github.com/sap/luigi/issues/2160#issuecomment-885539914) required **empty string** or **false** for disabling the tooltip. Here it seems like you are not checking for empty string, maybe should make it an **or** statement instead ? 

> If empty string or explicitly set to false, no title attribute will be rendered. [comment](https://github.com/sap/luigi/issues/2160#issuecomment-885539914)

Reply:
yep, something like that 👍